app/gateway/changelog.md

```diff
@@ -308,6 +308,22 @@ Kong Gateway version.
 * The function `kong.db.*.cache_key()` now falls back to `.id` if nothing from `cache_key` is found.
   [#8553](https://github.com/Kong/kong/pull/8553)
 
+### Known limitations
+
+* Kong Manager does not currently support the following features:
+  * Secrets management
+  * Plugin ordering
+  * Expression-based routing
+
+* Creating and updating Kong configuration for traditional mode is not allowed during migration from 2.x to 3.0.0.0.
+  * This is a known issue planned to be fixed in the next 2.8 release. If this is a requirement for upgrading, Kong operators should upgrade to that version before beginning a upgrade to 3.0.0.0.
+  * In traditional and hybrid mode Kong can be upgraded while continuing to proxy traffic.
```

Comment:
We are not going to support configuring while migrating. What we are trying to solve with 2.8.2 is we have downtime migrating with traditional mode(proxy port).